### PR TITLE
add environment variable `XMRSTAK_NOWIT`

### PR DESCRIPTION
**Pull request by psychocrypt**
- add documentation
- add environment variable `XMRSTAK_NOWAIT` to disable the dialog `Press any key` on windows.

After this PR and #305 is merged we can activate the appvoyer binary test again if we set the environment variable.

### CODE REVIEW

File: xmrstak/misc/console.cpp [hunk 1]
```diff
@@ -222,8 +222,13 @@ void printer::print_str(const char* str)
 #ifdef _WIN32
 void win_exit(size_t code)
 {
-	printer::inst()->print_str("Press any key to exit.");
-	get_key();
+	size_t envSize = 0;
+    getenv_s(&envSize, nullptr, 0, "XMRSTAK_NOWAIT");
```
Comment (fireice-uk) [inline xmrstak/misc/console.cpp]: Another space indent - please fix

**Comment by fireice-uk** [comment]
Are you going to re-enable appveyor in another pr? Note voyer != veyor :D

**Comment by psychocrypt** [comment]
Yes I will add a extra PR for appvoyer after this and #305 is merged.

**Comment by psychocrypt** [comment]
Shit I wrote it wrong, again :-)

**Pull request merged**